pipeline/buildpacks/0.1/README.md

```diff
@@ -0,0 +1,89 @@
+# Buildpacks
+
+This pipeline builds source into a container image using [Cloud Native Buildpacks](https://buildpacks.io). To do that, it uses [builders](https://buildpacks.io/docs/concepts/components/builder/#what-is-a-builder) to run buildpacks against your application source.
+
+> _**What are Cloud Native Buildpacks?**_
+> 
+> _Cloud Native Buildpacks are pluggable, modular tools that transform application source code into OCI images. They replace Dockerfiles in the app development lifecycle, and enable for swift rebasing of images and modular control over images (through the use of builders), among other benefits._
+
+## Dependencies
+
+- [`git-clone` task](https://github.com/tektoncd/catalog/tree/master/task/git-clone) 0.2 or newer
+- [`buildpacks` task](../../../task/buildpacks/) 0.3 or newer
+- [`buildpacks-phases` task](../../../task/buildpacks-phases/) 0.2 or newer
+
+## Compatibility
+
+- **Tekton** v0.12.1 and above
+- **[Platform API][platform-api]** 0.4
+
+[platform-api]: https://buildpacks.io/docs/reference/spec/platform-api/
+
+## Install
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/pipeline/buildpacks/0.1/buildpacks.yaml
+```
+
+## Workspaces
+
+ - **`source-ws`**: Location where source is stored. _(REQUIRED)_
+ - **`cache-ws`**: Location where cache is stored if CACHE_IMAGE is not provided. _(optional)_
+
+## Parameters
+
+ - **`BUILDER_IMAGE`**: The image on which builds will run (must include lifecycle and compatible buildpacks). _(REQUIRED)_
+ - **`TRUST_BUILDER`**: Whether the builder image is trusted. When false, each build phase is executed in isolation and credentials are only shared with trusted images. _(optional, default: "false")_
```

Comment:
should we document the accepted values over here so that user doesn't pass a wrong value when they want to opt-in for `TRUST_BUILDER`?

Reply:
I can add it to the description but I feel like it should be pretty obvious. Maybe it's just me 🤷‍♂️.

As a sidenote, **TLDR; It would be nice if there was a boolean/truthy type instead of just array and string**. I'm trying to [automate](https://github.com/buildpacks/tekton-integration/blob/main/scripts/generate-docs.sh#L70) as much of this as possible to minimize the burden of maintenance. As far as I'm aware, this sort of validation doesn't exist in the Tekton Pipeline and therefor it's not encoded into the API for me to derive this information when generating the docs.

Reply:
> I can add it to the description but I feel like it should be pretty obvious. Maybe it's just me .
> 
Okay...no worries we can skip this for now

> As a sidenote, **TLDR; It would be nice if there was a boolean/truthy type instead of just array and string**. I'm trying to [automate](https://github.com/buildpacks/tekton-integration/blob/main/scripts/generate-docs.sh#L70) as much of this as possible to minimize the burden of maintenance. As far as I'm aware, this sort of validation doesn't exist in the Tekton Pipeline and therefor it's not encoded into the API for me to derive this information when generating the docs.

Yeah I also agree that Tekton Pipelines should support other types as well like integer and boolean apart from array and string.